packages/eds-core-react/src/components/Divider/Divider.test.tsx

```diff
@@ -31,6 +32,10 @@ describe('Divider', () => {
     const { asFragment } = render(<Divider />)
     expect(asFragment()).toMatchSnapshot()
   })
+  it('Should pass a11y test', async () => {
+    const { container } = render(<Divider />)
```

Comment:
Maybe have a `p` before and after with some text to best test this component in terms of a11y as it can only really be tested correctly in context of "content it seperates"